plugins/modules/rulebook_activation.py

```diff
@@ -455,9 +542,16 @@ def main() -> None:
                 source_name=dict(type="str"),
             ),
         ),
-        swap_single_source=dict(type="bool", default=True),
-        log_level=dict(type="str", choices=["debug", "info", "error"], default="error"),
-        state=dict(choices=["present", "absent"], default="present"),
+        swap_single_source=dict(
+            type="bool",
+            default=True,
+            removed_in_version="3.0.0",
+            removed_from_collection="ansible.eda",
+        ),
+        log_level=dict(type="str", choices=["debug", "info", "error"]),
+        state=dict(
+            choices=["present", "absent", "enabled", "disabled"], default="present"
```

Comment:
As I elaborated in a previous comment, the new default should be "enabled" because the default of the api param "is_enabled" is True. I mean, by default the activation is created enabled.